pkgs/top-level/release.nix

```diff
@@ -98,6 +98,7 @@ let
               jobs.pkgs-lib-tests
               jobs.stdenv.x86_64-linux
               jobs.cargo.x86_64-linux
+              jobs.curl.x86_64-linux.tests.withCheck
```

Comment:
Why only x86_64-linux? Shouldn't this cover every platform or at the very least, all the platforms everyone depends on (being `(aarch64|x86_64)-(linux|darwin)`)?